modules/Core/includes/endpoints/GetAnnouncementsEndpoint.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * @param int $id NamelessMC ID of the user whose announcements to view
+ * 
+ * @return string JSON Array of latest announcements
+ */
+class GetAnnouncementsEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'getAnnouncements';
+        $this->_module = 'Core';
+        $this->_description = 'Return latest available announcements for the supplied user';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.